angular/lib/src/lib/pagination/pagination.component.ts

```diff
@@ -163,11 +173,14 @@ const defaultConfig: Partial<PaginationProps> = {
 };
 
 @Component({
-	selector: 'au-pagination',
+	// eslint-disable-next-line @angular-eslint/component-selector
```

Comment:
As we decided in Monday's meeting that this change of selector would be applied in all angular components, I think we could disable this rule with an override rule in the .eslintrc of the angular package, instead of adding this line for every component.

Reply:
will be done in another pr about eslint